From 1807aa5c70c4eb9fc7ca7de356236b05efa991cc Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Mon, 11 Jul 2022 11:05:45 +0800
Subject: [PATCH] [riscv64] Fix build error

Change-Id: I9cc2df6108b08aacf0b017f85128a9475022cedf
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3754286
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Commit-Queue: ji qiu <qiuji@iscas.ac.cn>
Reviewed-by: ji qiu <qiuji@iscas.ac.cn>
Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
Cr-Commit-Position: refs/heads/main@{#81625}
---
 src/codegen/riscv64/assembler-riscv64.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/codegen/riscv64/assembler-riscv64.h b/src/codegen/riscv64/assembler-riscv64.h
index 7250807c46d..6f85edbb76b 100644
--- a/src/codegen/riscv64/assembler-riscv64.h
+++ b/src/codegen/riscv64/assembler-riscv64.h
@@ -46,6 +46,7 @@
 #include "src/codegen/label.h"
 #include "src/codegen/riscv64/constants-riscv64.h"
 #include "src/codegen/riscv64/register-riscv64.h"
+#include "src/codegen/flush-instruction-cache.h"
 #include "src/objects/contexts.h"
 #include "src/objects/smi.h"
 
-- 
2.35.1

